variables.tf

```
@@ -58,24 +58,33 @@ variable "friendly_name_prefix" {
 }
 
 variable "instance_type" {
-  default = "m5.xlarge"
+  default     = "m5.xlarge"
+  description = "The type of EC2 instances to create."
+  type        = string
 }
 
 variable "network_id" {
   default     = ""
   description = "Network ID for existing network to deploy in."
```

Could we start this sentence the same way as the others? 
```suggestion
   description = "The network ID for existing network to deploy in."
```

---

I couldn't accept this suggestion, possibly because it was on an unchanged line 🤷🏻 Any way, this is fixed by 17e98b6. I took some additional liberty with the wording to align with other network and subnetwork variables.